crates/fluvio-smartengine/src/engine/wasmtime_engine/engine.rs

```diff
@@ -1,22 +1,17 @@
-use std::fmt::{self, Debug};
-
 use anyhow::Result;
```

Comment:
directory `wasmtime_engine` can be shortened to `wasmtime` since this is already engine

Reply:
I use `wasmtime_engine` because I doubt it will conflict with the `wasmtime` crate. But it seems it won't be a problem. I can use `self::wasmtime`

```
  --> crates/fluvio-smartengine/src/engine/mod.rs:17:9
   |
17 | pub use wasmtime::{SmartEngine, SmartModuleChainBuilder, SmartModuleChainInstance};
   |         ^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `wasmtime` could refer to a crate passed with `--extern`
   = help: use `::wasmtime` to refer to this crate unambiguously
note: `wasmtime` could also refer to the module defined here
  --> crates/fluvio-smartengine/src/engine/mod.rs:16:1
   |
16 | mod wasmtime;
   | ^^^^^^^^^^^^^
   = help: use `self::wasmtime` to refer to this module unambiguously

```